apiary-extensions-terraform/privileges-grantor/README.md

```diff
@@ -0,0 +1,52 @@
+# Overview
+
+Terraform module for setting up infrastructure for [Apiary Privilege Grantor](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-events/apiary-metastore-consumers/privileges-grantor).
+
+For more information please refer to the main [Apiary](https://github.com/ExpediaInc/apiary) project page.
+
+## Variables
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| instance\_name | Privilege Grantor instance name to identify resources in multi-instance deployments. | string | `""` | no |
+| memory | The amount of memory (in MiB) to be used by Lambda | string | `"512"` | no |
+| metastore\_events\_sns\_topic | SNS Topic for Hive Metastore events. | string | n/a | yes |
+| pg\_jars\_s3\_key | S3 key where zip file is located. | string | n/a | yes |
+| pg\_lambda\_bucket | Bucket where the Lambda zip can be found, for example 'bucket_name' (Note with s3://). Used together with `pg_jars_s3_key`. | string | n/a | yes |
+| pg\_lambda\_version | Version of the Privilege Grantor Lambda. | string | n/a | yes |
+| pg\_metastore\_uri | Thrift URI of the metastore to which Lambda will connect to. | string | n/a | yes |
+| security\_groups | Security groups in which Lambda will have access to. | list | n/a | yes |
+| subnets | Subnets in which Lambda will have access to. | list | n/a | yes |
+| tags | A map of tags to apply to resources. | map | `<map>` | no |
+
```

Comment:
Do we miss: `metastore_events_sns_topic`

Reply:
Ah, It was in my changes but not pushed. Updated now.

Reply:
it's required but not in the usage example

Reply:
added in the example.